william-wtr92/instamint
# feat: 🚀 like a comments, sort comments by likes

# I can like a comment, and see the publication's top comments

The goal of this pull request is to able users to like comments and see top comments in publications.

## Proposed Changes

Describe the changes you have made. For lists of significant changes, it may be helpful to structure them in list form:

- Added : 
  - Migrations : publications_comments_likes table and model to handle likes on publications
  - Server: Like comment service and API routes
- Fixed : 
  - Translations: errors translations for profile
- Updated : 
  - View : Publication modal actions components to handle like comments 

## Type of Change

What type of change are you making? (Check all that apply)


## How Has This Been Tested?

Explain the tests you conducted to validate your changes. Provide clear instructions so that the code review can
replicate your tests and confirm the results.

To test this feature, follow these instructions:

- Run at root `pnpm install`
- Do migrations with `pnpm run kn:latest`
- Run the api business, api files and the front end, then test the feature on any publication modal
- Create a publication and then send a comment, then you can try to like/unlike comments


## Checklist

Before submitting your PR, please check the appropriate boxes to confirm that you have taken the following actions:


## Screenshots

![image](https://github.com/william-wtr92/instamint/assets/90528753/79ae1e5f-a14f-4183-aedd-6b90d3ff692b)
